sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample5_OrchestrationPrediction.md

```diff
@@ -0,0 +1,116 @@
+# Analyze a conversation
+
+This sample demonstrates how to analyze an utterance using an Orchestration project. To get started, you'll need to create a Cognitive Language service endpoint and an API key. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md) for links and instructions.
+
+To analyze an utterance, you need to first create a `ConversationAnalysisClient` using an endpoint and API key. These can be stored in an environment variable, configuration setting, or any way that works for your application.
+
+```C# Snippet:ConversationAnalysisClient_Create
+Uri endpoint = new Uri("https://myaccount.api.cognitive.microsoft.com");
+AzureKeyCredential credential = new AzureKeyCredential("{api-key}");
+
+ConversationAnalysisClient client = new ConversationAnalysisClient(endpoint, credential);
+```
+
+Once you have created a client, you can call synchronous or asynchronous methods.
+
+## Synchronous
+
+```C# Snippet:ConversationAnalysis_AnalyzeConversationOrchestrationPrediction
+Response<AnalyzeConversationResult> response = client.AnalyzeConversation(
+    "DomainOrchestrator",
```

Comment:
Not that you have to change it now, but just to note what we did after the archboard review for QnA we'll need to do here post-beta: put the projectName and deploymentName in a ConversationsProject and pass that after the question as the second parameter. See `QuestionAnsweringClient.GetAnswers` at some point for an example. Again, though, you don't need to change it here (was just top of mind).

Reply:
Yea okay, makes sense. I've [created an issue](https://github.com/Azure/azure-sdk-for-net/issues/25027) for this so that we can keep track of it. Unfortunately, I still can't add the proper labels for it :(

Reply:
Similarly - but, again, not in the PR - we'll probably want to move all models to the root. See QnA's autorest.md sibling to this project for how I did that.